wrapanapi/systems/rhevm.py

```diff
@@ -649,6 +649,13 @@ def deploy(self, vm_name, cluster, timeout=900, power_on=True, **kwargs):
             vm.start()
         return vm
 
+    def list_disks(self, disk_id=None):
+        """Return a list of disks in the template"""
+        return [
+            disk.id if disk_id is True else disk.name
```

Comment:
what is expected of `disk_id` in input? Is it boolean? I'd suggest to add that in docblock.

Reply:
yes, it's boolean. I will add